akka-remote/src/main/scala/akka/remote/artery/BufferPool.scala

```diff
@@ -377,12 +377,11 @@ private[remote] final class EnvelopeBuffer(val byteBuffer: ByteBuffer) {
     // Write fixed length parts
     byteBuffer.put(VersionOffset, header.version)
     byteBuffer.put(FlagsOffset, header.flags)
-    byteBuffer.putLong(UidOffset, header.uid)
-    byteBuffer.putInt(SerializerOffset, header.serializer)
-
```

Comment:
changes here are not needed, but I found it easier to read this if the fields were listed in the same order as they are in the header